de.tudarmstadt.ukp.dkpro.argumentation.nio/src/main/java/de/tudarmstadt/ukp/dkpro/argumentation/annotations/AnnotatedDocument.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) the Department of Informatics, Technische Universität Darmstadt. All Rights Reserved.
+ *
+ * Unauthorized distribution of this file via any medium is strictly prohibited.
+ */
+package de.tudarmstadt.ukp.dkpro.argumentation.annotations;
```

Comment:
> Unauthorized distribution of this file via any medium is strictly prohibited.

This is an Apache-licensed open source project. Please do not contribute any things that are confidential and non-redistributable! Ensure that contributed files have a proper license header corresponding to the project license.